deploy/updater/updater.py

```
@@ -1033,16 +1082,20 @@ async def process_alerts():
             logger.error(f"❌ process_alerts error: {str(e)}")
             logger.debug(f"❌ Повний стек помилки:", exc_info=True)
 
-    # Основний цикл очікування повідомлень з Pub/Sub
+    # Основний цикл очікування повідомлень з Pub/Sub (з debounce)
+    debouncer = Debouncer(fusion_alerts_debounce)
```

Ruff (0.15.2)

[error] Local variable `debouncer` is assigned to but never used

Remove assignment to unused variable `debouncer`

(F841)